1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md

```diff
@@ -0,0 +1,189 @@
+# Pengantar Bahasa Pemrograman dan Alat Keterampilan
+
+Pelajaran ini mencakup dasar-dasar bahasa pemrograman. Topik yang dibahas di sini berlaku untuk sebagian besar bahasa pemrograman modern saat ini. Di bagian 'Alat Keterampilan', Anda akan belajar tentang perangkat lunak yang berguna yang membantu Anda sebagai pengembang.
+
+![Pemrograman Intro](webdev101-programming.png)
+> Catatan sketsa oleh [Tomomi Imura](https://twitter.com/girlie_mac)
+
+## Kuis Pra-Kuliah
+[Kuis pra-kuliah](.github/pre-lecture-quiz.md)
```

Comment:
Is it possible to change this into [Kuis pra-kuliah](../.github/pre-lecture-quiz.md)?
Or do we need to create .github folder for translations?

Reply:
What do you think? @jlooper

Reply:
I'll add .github folder inside translations folder for now.